src/oatpp/network/monitor/ConnectionMaxAgeChecker.hpp

```diff
@@ -46,7 +48,7 @@ class ConnectionMaxAgeChecker : public MetricsChecker {
 
   std::vector<oatpp::String> getMetricsList() override;
 
-  std::shared_ptr<StatCollector> createStatCollector(const oatpp::String& metricName) override;
+  std::shared_ptr<StatCollector> createStatCollector(const oatpp::String& metricName) override GPP_ATTRIBUTE(noreturn);
```

Comment:
Doesn't look like a `noreturn` function to me.
Please give more details.

Reply:
The implementation throws a std::runtime_error

Reply:
Yes, you are correct. It always throws.